corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -435,6 +437,26 @@ def last_sync_log(self):
         else:
             return None
 
+    def _check_for_date_opened_bug(self, sync_log):
+        introduced_date = datetime(2016, 7, 19, 19, 15)
+        reverted_date = datetime(2016, 7, 20, 9, 15)  # date bug was reverted on HQ
+        resolved_date = datetime(2016, 7, 21, 0, 0)  # approximate date this fix was deployed
```

Comment:
@calellowitz made a good point that people who sync multiple times between when this gets deployed and midnight will consistently get 412's. Just noting that as a potential downside.